module/os/linux/zfs/policy.c

```diff
@@ -46,17 +47,20 @@ static int
 priv_policy_ns(const cred_t *cr, int capability, int err,
     struct user_namespace *ns)
 {
-	if (cr != CRED() && (cr != kcred))
-		return (err);
+	const cred_t *old =
+	    (cr != CRED() && cr != kcred) ? override_creds(cr) : NULL;
```

Comment:
Would be good to update the comment above.

Reply:
Oh, completely missed that on my todo list. Thanks.